src/Shouldly.Tests/ShouldThrow/ActionDelegateScenario.cs

```diff
@@ -18,7 +18,7 @@ protected override string ChuckedAWobblyErrorMessage
         protected override void ShouldPass()
         {
             var ex = Should.Throw<NotImplementedException>(() => { throw new NotImplementedException(); });
-            ex.ShouldBeOfType<NotImplementedException>();
+            ex.ShouldBeOfType<NotImplementedException>(() => "Some additional context");
```

Comment:
All the ShouldThrow tests like this one do not need to pass the additional context. Could you remove it from this class and the other 4 which are testing Should.Throw?